TESTS/netsocket/README.md

```diff
@@ -403,6 +411,53 @@ Wi-Fi tests require some more configuration, so for Wi-Fi purposes, the `mbed_ap
 }
 ```
 
+Cellular tests require some more configuration, so for cellular purposes, the `mbed_app.json` might look like this:
```

Comment:
Could also refer to `CellularInterface.json` with a spell, e.g.: `mbed test -n tests-netsocket-udp --app-config=tools/test_configs/CellularInterface.json -vv`

Reply:
Netsocket test README.md already has a reference to these example config files:

`Please, see `mbed-os/tools/test_configs` folder for examples for different technologies.`

I updated it a bit to mention about different technologies.